sonic_psu/psu_base.py

```diff
@@ -26,10 +26,22 @@ def get_num_psus(self):
     @abc.abstractmethod
     def get_psu_status(self, index):
         """
-        Retrieves the oprational status of power supply unit (PSU) defined
+        Retrieves the operational status of power supply unit (PSU) defined
                 by index <index>
 
         :param index: An integer, index of the PSU of which to query status
```

Comment:
Is it zero-based index? I think this info should be here.

Reply:
Added more details